### PR TITLE
[zh] Resync localization strings for zh-cn

### PR DESCRIPTION
**Pull request by tengqm**
closes: #36758

### CODE REVIEW

File: data/i18n/zh-cn/zh-cn.toml [hunk 1]
```diff
@@ -28,11 +35,38 @@ other = "Twitter"
...
+
+[cve_table]
+other = "Kubernetes CVE 列表"
+
+[cve_url]
+other = "CVE URL"
+
 [deprecation_title]
-other = "您正在查看 Kubernetes 版本的文档："
+other = "你正在查看的文档是 Kubernetes 版本："
```
Comment (windsonsea) [inline data/i18n/zh-cn/zh-cn.toml:66]:
```suggestion
other = "你正在查看的文档适用于 Kubernetes 版本："
```

**Review by windsonsea — COMMENTED** [review]
A little strange: `文档`是`版本`

File: data/i18n/zh-cn/zh-cn.toml [hunk 2]
```diff
@@ -28,11 +35,38 @@ other = "Twitter"
...
+other = "url"
+
+[cve_summary]
+other = "问题描述"
+
+[cve_table]
+other = "Kubernetes CVE 列表"
+
+[cve_url]
+other = "CVE URL"
```
Comment (Sea-n) [inline data/i18n/zh-cn/zh-cn.toml:63]: Seems this term is not used at this time.

```suggestion
other = "CVE 網址"
```

File: data/i18n/zh-cn/zh-cn.toml [hunk 3]
```diff
@@ -28,11 +35,38 @@ other = "Twitter"
 [community_youtube_name]
 other = "YouTube"
 
+[cve_id]
+other = "CVE ID"
+
+[cve_issue_url]
+other = "CVE GitHub Issue URL"
```
Comment (Sea-n) [inline data/i18n/zh-cn/zh-cn.toml:42]: https://kubernetes.io/zh-cn/docs/reference/issues-security/official-cve-feed/

```suggestion
other = "CVE GitHub Issue 網址"
```

File: data/i18n/zh-cn/zh-cn.toml [hunk 4]
```diff
@@ -218,6 +249,9 @@ other = "选项"
 [outdated_blog__message]
 other = "这篇文章已经一年多了，较旧的文章可能包含过时的内容。请检查从发表以来，页面中的信息是否变得不正确。"
 
+[post_create_issue]
+other = "登记一个问题"
```
Comment (Sea-n) [inline data/i18n/zh-cn/zh-cn.toml:253]: It's duplicated with upstream, I think we should remove it from `en.toml` and use Docsy version.

https://github.com/google/docsy/blob/main/i18n/zh-cn.toml#L44

File: data/i18n/zh-cn/zh-cn.toml [hunk 5]
```diff
@@ -235,6 +269,21 @@ other = "(发布日期: "
 [release_date_format]
 other = "2006-01-02"
 
+# Deprecated. Planned for removal in a future release.
+# Use [release_full_details_initial_text] instead.
+[release_complete]
+other = "完整的"
```
Comment (Sea-n) [inline data/i18n/zh-cn/zh-cn.toml:275]: This term is never used, we should just remove it from `en.toml`.

File: data/i18n/zh-cn/zh-cn.toml [hunk 6]
```diff
@@ -1,6 +1,13 @@
 # i18n strings for the Chinese version of the site (https://kubernetes.io/zh-cn/)
 # 注意：修改此文件时请维持字符串名称的字母顺序并与英文版保持一致
 
+# Avoid using conjunction_1.
+# Must match the context in layouts/shortcodes/release-data.html
+# Appears on https://kubernetes.io/releases/
+# For example the "and" in "Complete 1.25 Schedule and Changelog"
```
Comment (rolfedh) [inline data/i18n/zh-cn/zh-cn.toml:7]: 将这些说明本地化是否有帮助？
Would it help to localize these instructions?

File: data/i18n/zh-cn/zh-cn.toml (hunk 5 above)
Comment (rolfedh) [reply data/i18n/zh-cn/zh-cn.toml:275]: You're right, @Sea-n. I've created https://github.com/kubernetes/website/issues/36839 to address this.

Comment (rolfedh) [reply data/i18n/zh-cn/zh-cn.toml:275]: 将这些评论本地化是否有帮助？
Would it help to localize these comments?

File: data/i18n/zh-cn/zh-cn.toml [hunk 7]
```diff
@@ -235,6 +269,21 @@ other = "(发布日期: "
 [release_date_format]
 other = "2006-01-02"
 
+# Deprecated. Planned for removal in a future release.
+# Use [release_full_details_initial_text] instead.
+[release_complete]
+other = "完整的"
+
+# Replace [release_complete] with [release_full_details_initial_text]
```
Comment (rolfedh) [inline data/i18n/zh-cn/zh-cn.toml:277]: 将此评论本地化是否有帮助？
Would it help to localize this comment?

Comment (tengqm) [reply data/i18n/zh-cn/zh-cn.toml:277]: There is no need to localize the comments because the comments are not user facing.

Comment (rolfedh) [reply data/i18n/zh-cn/zh-cn.toml:277]: Okay, so there's no need to localize for _internal_ users, such as folks working on the localized content.

File: data/i18n/zh-cn/zh-cn.toml (hunk 5 above)
Comment (rolfedh) [reply data/i18n/zh-cn/zh-cn.toml:275]: We can mark this as resolved.

File: data/i18n/zh-cn/zh-cn.toml (hunk 3 above)
Comment (tengqm) [reply data/i18n/zh-cn/zh-cn.toml:42]: URL is unlikely to be misunderstood, right?

File: data/i18n/zh-cn/zh-cn.toml (hunk 2 above)
Comment (tengqm) [reply data/i18n/zh-cn/zh-cn.toml:63]: We can revisit this one when the upstream English version is changed.

**Comment by Sea-n** [comment]
I think my previous comment are all non-blocking issues.

/approve

**Comment by windsonsea** [comment]
Thanks.

/lgtm

**Pull request merged**